.github/workflows/main.yml

```diff
@@ -100,7 +100,9 @@ jobs:
         run: pipx run coverage xml --ignore-errors
       - name: Publish coverage
         if: hashFiles('coverage.xml') != ''  # Rudimentary `file.exists()`
-        uses: codecov/codecov-action@v3
+        uses: codecov/codecov-action@v4
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Please, use an input instead of an env var.

Reply:
I followed codecov's documentation. Do you have an example of how this should be done instead to be improved?

Reply:
https://github.com/codecov/codecov-action#usage

Reply:
```suggestion
        with:
          token: ${{ secrets.CODECOV_TOKEN }}
```

Reply:
I guess the doc page I was looking at was outdated as it only offered using `env` rather than passing it as an input